Error in Travis & AppVeyor 👀

```rust
error[E0432]: unresolved import `headers_ext::Basic`
  --> components/net/tests/http_loader.rs:19:46
   |
19 |     AccessControlAllowOrigin, Authorization, Basic, ContentLength, Date, Host, Origin,
   |                                              ^^^^^ no `Basic` in the root
error: aborting due to previous error
```

--- a/components/net/Cargo.toml
+++ b/components/net/Cargo.toml
@@ -14,7 +14,7 @@ test = false
 doctest = false
 
 [dependencies]
-base64 = "0.9"
+base64 = "0.10.1"
 brotli = "3"
 bytes = "0.4"
 cookie_rs = {package = "cookie", version = "0.11"}

--- a/components/net/data_loader.rs
+++ b/components/net/data_loader.rs
@@ -50,7 +50,7 @@ pub fn decode(url: &ServoUrl) -> Result<DecodeData, DecodeError> {
             .into_iter()
             .filter(|&b| b != b' ')
             .collect::<Vec<u8>>();
-        match base64::decode(&bytes) {
+        match base64::decode_config(&bytes, base64::STANDARD.decode_allow_trailing_bits(true)) {
             Err(..) => return Err(DecodeError::NonBase64DataUri),
             Ok(data) => bytes = data,
         }

--- a/components/script/Cargo.toml
+++ b/components/script/Cargo.toml
@@ -32,7 +32,7 @@ tinyfiledialogs = "3.0"
 [dependencies]
 app_units = "0.7"
 backtrace = {version = "0.3", optional = true}
-base64 = "0.9"
+base64 = "0.10.1"
 bitflags = "1.0"
 bluetooth_traits = {path = "../bluetooth_traits"}
 byteorder = "1.0"

--- a/components/script/dom/window.rs
+++ b/components/script/dom/window.rs
@@ -509,12 +509,9 @@ pub fn base64_atob(input: DOMString) -> Fallible<DOMString> {
         return Err(Error::InvalidCharacter);
     }
 
-    match base64::decode(&input) {
-        Ok(data) => Ok(DOMString::from(
-            data.iter().map(|&b| b as char).collect::<String>(),
-        )),
-        Err(..) => Err(Error::InvalidCharacter),
-    }
+    let data = base64::decode_config(&input, base64::STANDARD.decode_allow_trailing_bits(true))
+        .map_err(|_| Error::InvalidCharacter)?;
+    Ok(data.iter().map(|&b| b as char).collect::<String>().into())
 }
 
 impl WindowMethods for Window {

--- a/components/webdriver_server/Cargo.toml
+++ b/components/webdriver_server/Cargo.toml
@@ -11,7 +11,7 @@ name = "webdriver_server"
 path = "lib.rs"
 
 [dependencies]
-base64 = "0.9"
+base64 = "0.10"
 cookie = "0.11"
 crossbeam-channel = "0.3"
 euclid = "0.19"

